Cargo.toml

```
@@ -15,6 +15,8 @@ readme = "README.md"
 [dependencies]
 serde = { version = "1.0.183", features = ["derive"], optional = true}
 thiserror = "1.0"
+mac_addr = { package = "pnet_base", version = "0.35.0"} # It only contains the code for MacAddr.
```

@mokeyish I wonder how this differs from importing the crate as:


```suggestion
pnet_base = "0.35"
```